src/App.js

```diff
@@ -1,12 +1,25 @@
 import MainPage from "./modules/main-page";
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from "react-router-dom";
```

Comment:
it is better if we organize the imports, firstly the library imports, then our local ones